pkgs/tinydm-jovian/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, shellcheck }:
+
+stdenv.mkDerivation rec {
+  pname = "tinydm-jovian";
+  version = "1.1.3";
+
+  src = ./tinydm-run-session.sh;
```

Comment:
Should we consider cloning and syncing the actual repo to better track our changes?

New repos in GitHub aren't expensive. It should make it easier to rebase changes with upstream, and also better keep obvious attribution through the history.

Reply:
It's now pushed to https://github.com/Jovian-Experiments/tinydm-jovian.